ocs_ci/ocs/constants.py

```diff
@@ -377,6 +377,10 @@
 CEPHFS_INTERFACE = 'cephfs'
 RAW_BLOCK_DEVICE = '/dev/rbdblock'
 
+# Constant values for IOPS and Throughput
+IOPS_FOR_1TiB_OSD = 3000
```

Comment:
need to clarify that those numbers are for AWS m5.4xlarge instances.

Reply:
This value is set by considering 3 things in mind
1. gp2 interface
2. EBS volumes
3. EC2 instances.